spec/labor/markdown_parser_spec.rb

```diff
@@ -263,7 +263,7 @@ def generate_and_parse_markdown(raw_markdown)
   context "when a colon emoji is used" do
     it "doesn't change text in codeblock" do
       result = generate_and_parse_markdown("<span>:o:<code>:o:</code>:o:<code>:o:</code>:o:<span>:o:</span>:o:</span>")
-      expect(result).to include("<span>⭕️<code>:o:</code>⭕️<code>:o:</code>⭕️<span>⭕️</span>⭕️</span>")
+      expect(result).to include("<span>⭕<code>:o:</code>⭕<code>:o:</code>⭕<span>⭕</span>⭕</span>")
```

Comment:
is it just me or...are these two lines the same? 🤔

Reply:
not according to git diff :D I think they updated the icons